include/common/s1ap_structs.h

```diff
@@ -172,7 +172,7 @@ typedef struct targetId{
         struct TAI selected_tai;
 }targetId;
 
-#define TRANS_CONT_SIZE 2700
+#define TRANS_CONT_SIZE 3000
```

Comment:
should we just make it 4k ?

Reply:
we can update if needed. currently it is able to handle the src to target container size for iPhone and Samsung.